.github/workflows/stale.yml

```
@@ -0,0 +1,24 @@
+name: "Stale issues and pull requests"
+on:
+  schedule:
+  - cron: "10 3 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        days-before-stale: '-1'
+        days-before-close: '90'
```

```suggestion
        days-before-close: '90'
        only-labels: 'waiting-response'
```

We only want to consider stale issues/PRs which maintainers had the chance to review and label - as it was in hashibot.